### ISSUE TITLE
Check version failure

### ISSUE THREAD

**Issue by gui28347** [winsiderss/systeminformer#1433]
### Brief description of your issue

Impossible to check for updates

### Steps to reproduce (optional)

Click Help - Check for updates

### Expected behavior (optional)

See if there is an update or not

### Actual behavior (optional)

Error is returned:
![image](https://user-images.githubusercontent.com/8279438/196328203-2116e2e5-4aa9-4da0-a646-df82cfdfd136.png)

### Environment (optional)

```shell
System Informer - nightly version
Windows 11
https://systeminformer.sourceforge.io/nightly.php?update
{
"version":"",
"updated":"",
"commit":"",
"bin_length":"",
"bin_hash":"",
"bin_sig":"",
"bin_url":"",
"setup_length":"",
"setup_hash":"",
"setup_sig":"",
"setup_url":"" 
}
```

**Comment by jxy-s** [comment]
RTFM please and thank you: https://github.com/winsiderss/systeminformer/issues/1286

**Comment by dmex** [comment]
The nightly download page here: https://processhacker.sourceforge.io/nightly.php

**Comment by gui28347** [comment]
> The nightly download page here: https://processhacker.sourceforge.io/nightly.php

The source code is referencing :   https://processhacker.sourceforge.io/nightly.php?updates

It was returning blank values till moments ago. Now it is returning values
{
"version":"3.0.4953",
"updated":"2022-06-13T04:49:58.7755638+00:00",
"commit":"37e0466fab1277524ed20b9cad0138c06680f961",
"bin_length":"7021322",
"bin_hash":"A5C3EECB80AFC94CD28511DBCA55B297DBD414EC061D309B1777730BBD8933EA",
"bin_sig":"e63fe71ef0e9ce0ea170f1389032b255290be9e5a1b5f7f22f8683a6bf15373bc8ae4a47384f0cb14e93ae0dbc92d06365cef46f1d85d51c265a7b905400e149",
"bin_url":"https://github.com/ProcessHackerRepoTool/nightly-builds-mirror/releases/download/3.0.4953/processhacker-3.0.4953-bin.zip",
"setup_length":"9827840",
"setup_hash":"4EF376A8EE7308806930BCFBC0CADC055E456413F52E8CE3027274CCEC3B573B",
"setup_sig":"72350cd556d961a693833f51ec86d65c20bb6c42a07029b4574cfed640e15e1c9d2eb7792677317ce52f29ea27495fb189c00b3f1b80b952bac91e93e888d38f",
"setup_url":"https://github.com/ProcessHackerRepoTool/nightly-builds-mirror/releases/download/3.0.4953/processhacker-3.0.4953-setup.exe" 
}

**Comment by dmex** [comment]
> The source code is referencing

Yes. That server isn't live for the public yet... things are being tested as part of the migration #1286
